test/Ed25519VerificationKey2020.spec.js

```diff
@@ -45,9 +43,22 @@ describe('Ed25519Signature2020', () => {
       .buildDocumentLoader();
   });
 
-  describe('constructor', () => {
-    it('should exist', async () => {
-      Ed25519Signature2020.should.exist;
+  describe('exports', () => {
+    it('it should have proper exports', async () => {
+      should.exist(Ed25519Signature2020);
+      should.exist(suiteContext);
+      suiteContext.should.have.keys([
+        'appContextMap',
+        'constants',
+        'contexts',
+        'documentLoader',
+      ]);
+      Ed25519Signature2020.CONTEXT_URL.should.exist;
```

Comment:
I think there's a problem with this syntax, which is that if `CONTEXT_URL` does not exist, the error is `could not access.should.of undefined`, instead of a proper assertion error.  I think the preferred syntax is demonstrated on L49
```suggestion
      should.exist(Ed25519Signature2020.CONTEXT_URL);
```

Reply:
Got it, thanks! (I'm less familiar w the `should` syntax.)